Skid steering articulation test (#12)
RobotecAI/o3de-ros2-gem-testing: RobotControlTest/06_CONTROL_TWIST_DRIVE_ARTICULATIONS/CONTROL_TWIST_DRIVE_TESTCASE.md

# Test Goal

- Check if ROS2 `geometry_msgs/msg/Twist` messages are properly interpreted by the skid steering vehicle model

> # Test Perquisite

Reviewer: ```suggestion
# Test Prerequisites
```

- Empty default Level
- ROS2 Gem activated

> - RosRobotSample Gem activated 

Reviewer: Is this gem still needed? You attached a prefab.
```suggestion
```

- O3DE Editor running
- Physx 5 enabled

# Steps

## Step 1

Copy the file: [rosbot_xl_articulations.prefab](../Assets/rosbot_xl_articulations.prefab) into the Assets folder of your project and instantiate it. Move the robot to be located above the ground plane.

> > Note : currently robot mass is set to very large value, to mitigate some instabilities.
> > TODO: change the ROSbot to something that does not relay on additional gem.

Reviewer: Is `TODO` still valid? There is a prefab attached in this PR.

```suggestion
> Note: currently robot mass is set to a very large value, to mitigate some instabilities.
> TODO: change the ROSbot to something that does not rely on additional Gems.
```

### Expected result

Your level should look like this:
![](images/control_initial_setup.png)

## Step 2

> Press `Ctrl+G` to enter game mode.

Reviewer: ```suggestion
Press `Ctrl+G` to enter the game mode.
```
